apps/frontend/src/api/util.ts

```diff
@@ -16,15 +16,18 @@ export const useFetchOptions = ({
     options: { headers, ...options },
     ...args
   }) => {
-    if (!isAuthenticated) {
-      throw new Error('No authorization bearer token available');
-    }
+    const authHeaders = isAuthenticated
```

Comment:
I think we should hide this behind an option. Reason why I originally made it throw is if you accidentally call it without waiting for user authorization and expect it to set the header, it'll be a very confusing hard to debug failure.

Reply:
We don't need to authenticate the request that lead me to do this change, so I'm reverting it.